code/src/layouts/app-header/components/header/Header.scss

```diff
@@ -0,0 +1,51 @@
+.wrapper {
+  padding: 10px 20px;
+  background-color: #f8f9fa;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);

```

Comment:
Same problem. For box shadow makes sense to keep our shadows in design system. Please take a look how it is implemented on SpaceToStudy project in file `_variables.scss`